### PR TITLE
Add channel as the return path from Parse() to the caller.

### PR DESCRIPTION
**Pull request by morrowc**
Move the Parse function from emitting a []Records to filling a channel with Records.

callers can now just open the channel and read as soon as records are available.
This should enable a caller to open many files at once and parse the data into Records to be consumed asynchronously.

### CODE REVIEW

**Comment by morrowc** [comment]
Travis build/test results of this pull request:
  https://travis-ci.org/morrowc/contrib/builds/494360868

**Comment by morrowc** [comment]
https://travis-ci.org/morrowc/contrib/builds/494362676

actual travis build url.

File: rpsl-parser/irr_test.go [hunk 1]
```diff
@@ -746,10 +752,29 @@ func TestParse(t *testing.T) {
 		}
 
 		r := NewReader(fd)
-		got, err := Parse(r)
-		if err == io.EOF {
+		rc := make(chan *Record, 10)
```
Comment (aashaikh) [inline rpsl-parser/irr_test.go:750]: may want to think about putting an example snippet in the readme (or in an example file) just to show a sample client for the reader ...

**Review by aashaikh — APPROVED** [review]
LGTM, minor suggestion.

File: rpsl-parser/irr_test.go (hunk 1 above)
Comment (morrowc) [reply rpsl-parser/irr_test.go:750]: Oh, this is a good point. 
I'll update the readme to include a quick client.

or rather edit the existng example :)

Comment (morrowc) [reply rpsl-parser/irr_test.go:750]: done

**Pull request merged**